### ISSUE TITLE
use of f-string

### ISSUE THREAD

**Issue by sandrotosi** [sandrotosi/puddletag#60]
we should move away from `format()` or string concatenation and embrace the beauty of f-strings throughout puddletag codebase

i believe this will reduce complexity and increase readibility

**Comment by pedrobaeza** [comment]
f-string are only available since P3.6, so compatibility will be reduced (I think Ubuntu 16.04 didn't ship this Python version for example). If it's admissible or not is what has to be judged.

**Comment by sandrotosi** [comment]
i dont think we should limit development to support a distribution released 4+ years ago. We already started using f-strings in puddletag, so the support will be for distros with py3.6+

**Comment by pedrobaeza** [comment]
Ok, it was just a compatibility comment for having into account. I can agree we should move forward.
